### DISCUSSION TITLE
TFT 35 Compatibility

### DISCUSSION THREAD

**Discussion by FlyBoy33** [mriscoc/Ender3V2S1#960] [Q&A]
Good Day,
I am using an SKR Mini E3V3.0 on an Ender 3V2.
I am using a BTT TFT35 screen.
What configuration do I need in order to use my TFT35? Is it compatable?

Any help would greatly be appreciated! I am fairly green so please be detailed in your response!

**Comment by Crazy-Charles** [comment]
Its support in the Marlin firmware but not in this one.

**Reply by FlyBoy33** [reply]
Sorry for my stupidity. Are you saying I cannot get the professional grade firmware on my skr mini, and still use the TFT35? Or are you saying that i need to download different firmware?
Am I able to professional firmware on my MINI SKR and still use my TFT35? If so, can you please elaborate on what firmware i need?

Thank you so much in advance!

**Reply by Crazy-Charles** [reply]
The combination of the SKR mini and TFT35 is not available on the "the professional grade firmware" so no, you cant use the TFT35 display (this is the un-supported part), the Marlin firmware does support both.

**Reply by FlyBoy33** [reply]
Does the Marlin version offer the advanced features such as mesh building, z offset adjustment during print, etc? I'm appreciating your help!

**Reply by Crazy-Charles** [reply]
Yes it offers all of those https://github.com/MarlinFirmware

**Reply by FlyBoy33** [reply]
So do i just download the Marlin Bugfix 2.0 and install that for the SKR? Because that didnt work. Screen says no printer connected.
